spaces/hurricane/index.json

```
@@ -0,0 +1,20 @@
+{
+  "name": "Hurricane.Finance",
+  "network": "42",
+  "symbol": "CAT5",
+  "skin": "hurricane",
```

You will need to add domain field to make your subdomain work. Related to #478  
```suggestion
  "skin": "hurricane",
  "domain": "gov.hurricane.finance",
```